client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -602,8 +602,7 @@ public void testGetMapping() throws IOException {
             // end::get-mappings-request
 
             // tag::get-mappings-request-masterTimeout
-            request.masterNodeTimeout(TimeValue.timeValueMinutes(1)); // <1>
-            request.masterNodeTimeout("1m"); // <2>
+            request.setMasterTimeout(TimeValue.timeValueMinutes(1)); // <1>
```

Comment:
Out of curiosity, did you remove the second setter on purpose? Might be a useful one in this case, but I'm also fine with keeping the new request lean from the start. Just wanted to know.

Reply:
I went with the strategy we seem to be using for other HLRC requests, where we add these setters by extending `TimedRequest`. That parent class only includes one setter, and doesn't accept the string-based time representation.